backend/src/layer/data/database/mealplan_management.rs

```diff
@@ -1313,4 +1201,94 @@ mod test {
         assert_eq!(price.price_guest, record.price_guest as u32);
         assert_eq!(price.price_pupil, record.price_pupil as u32);
     }
+
+    async fn get_env_info(pool: &PgPool, food_id: Uuid) -> ParseEnvironmentInfo {
+        ParseEnvironmentInfo::from_row(
+            &sqlx::query(&format!(
+                r#"SELECT * FROM food_env_score WHERE food_id = '{food_id}'"#
```

Comment:
just don't... not even in tests...
Why not useing bind?

Reply:
I don't get the Problem. What should I "bind" there?

Reply:
You are formatting a value (input) into the sql string. If this would not be a test and maybe a string input, this would create a SQL-injection vulnerability.
To do it properly, you would add a placeholder `#1`